### DISCUSSION TITLE
When copy pasting elements, the pasted content needs to appear when the mouse pointer is.

### DISCUSSION THREAD

**Discussion by arun-chief** [jgraph/drawio#4292] [General]
When pasting the contents selected using the mouse, the copied content is appearing just near to the element I copied. 

I want the elements to be pasted, when the mouse is currently pointing.

Very difficult to complete the task efficiently due to lack of this. Most of the time, the pasted content is appearing over other content, which make it very difficult to select again and move it. If there is multiple elements, then it is even more difficult.

![image](https://github.com/jgraph/drawio/assets/16814342/0e23fa02-7cb7-426f-b710-96cd9d96fd46)

![image](https://github.com/jgraph/drawio/assets/16814342/1ce994bd-d0cb-4228-8374-134541bbfc72)

**Comment by davidjgraph** [comment]
Duplicate #4053. Please fill in the issue template and search before posting.
